mailing_list/static/mailing_list/js/app.js

```diff
@@ -23,16 +23,29 @@
         ml.isUpdating = false;
         ml.primarySectionLists = [];
         ml.otherSectionLists = [];
+        ml.updatedAccessLevel = '';
```

Comment:
I'm not sure I understand the use of ml.updatedAccessLevel, why not just use the existing property on the list?